django_countries/templatetags/countries.py

```diff
@@ -1,10 +1,11 @@
-from django import template
+from django import VERSION, template
 from django_countries.fields import Country
 
 
 register = template.Library()
 
+simple_tag = register.assignment_tag if VERSION < (1, 9) else register.simple_tag
```

Comment:
`django.template.Library.assignment_tag` is pending deprecation warning in Django 1.9.